src/main/java/com/amogoscode/groupe/ebankingsuite/User/User.java

```diff
@@ -0,0 +1,136 @@
+package com.amogoscode.groupe.ebankingsuite.User;
```

Comment:
package should be lower case 
```
package com.amogoscode.groupe.ebankingsuite.user;
```

Reply:
groupe -> group

Reply:
changes have been pushed in the new commit